scripts/apps/search/MultiImageEdit.ts

```
@@ -44,7 +44,7 @@ export function MultiImageEditController(
         const updatedImages = imagesOriginal.map((image) => {
             const existingImage = $scope.images.find(({_id}) => _id === image._id);
 
-            return existingImage != null ? existingImage : {...image, ...$scope.metadata};
+            return existingImage != null ? existingImage : {...image, ...pickBy($scope.metadata)};
```

Are you picking everything? Do you need to pick at all then?

---

I'm picking everything that's not empty

---

Can you then use `(item) => item != null` predicate. It's sneaky that lodash is silently ignoring `null`.

---

added explicit check